mica-webapp/src/main/resources/i18n/en.json

```diff
@@ -819,6 +819,8 @@
   "user": "User",
   "users": "Users",
   "website": "Website",
+  "unsaved-title": "Unsaved Changes",
+  "unsaved-prompt": "Unsaved changes will be lost. Are you sure you'd like to leave this page?",
```

Comment:
May be a better message, a suggestion: Are you sure you want this page? Unsaved changes will be lost.